src/officehours_api/backends/inperson.py

```diff
@@ -1,11 +1,13 @@
 from django.contrib.auth.models import User
 
-from officehours_api.backends.backend_dict import BackendDict
+from officehours_api.backends.types import BackendDict, IMPLEMENTED_BACKEND_NAME
 
 
 class Backend:
-    name = 'inperson'
+    name: IMPLEMENTED_BACKEND_NAME = 'inperson'
     friendly_name = "In Person"
+    enabled = True
```

Comment:
Nitpick, but it might make a bit more sense to handle this the same way as the other backends, since it would be more consistent and `inperson` could potentially be disabled by providing a different `settings.py`.

Reply:
Sure, no problem, I can do that.

Reply:
Done. See https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/305/commits/6b9802a2dc5156ed5c41299439e05daf8d78866a